data/manager_routes.php

```diff
@@ -9,6 +9,7 @@
 $user 					 = '(?P<user_uid>([a-zA-Z0-9-_]+))';
 $subscription 			 = '(?P<subscription_id>([0-9]+))';
 $event 			 		 = '(?P<event_id>([0-9]+))';
+$event_datetime			 = '(?P<event_datetime_id>([0-9]+))';
```

Comment:
This is pretty pesky of me, but there are a lot of alignment issues in this file.